extension/popup.html

```diff
@@ -46,23 +46,23 @@ <h1 class="settings-hl">Settings</h1>
       </sign-up-panel>
 
       <main-panel class="signed-in-panel hidden">
-          <p class="ff-Met aliases-remaining text-center remaining-plural">You have <span class="num-aliases-remaining"></span><span class="ff-Met max-num-aliases"></span></span> relay addresses remaining</p>
-          <onboarding-panel>
-            <img class="onboarding-img">
-            <h1 class="onboarding-h1"></h1>
-            <p class="onboarding-p"></p>
+        <p class="fx-relay-font-met aliases-remaining remaining-plural">You have <span class="num-aliases-remaining"></span><span class="fx-relay-font-met max-num-aliases"></span></span> relay addresses remaining</p>
+        <onboarding-panel>
+          <img class="onboarding-img">
+          <h1 class="onboarding-h1"></h1>
+          <p class="onboarding-p"></p>
 
-            <div class="onboarding-pagination">
-              <button class="panel-nav previous-panel" data-direction="-1" aria-label="Show previous panel"></button>
-              <span class="panel-num ff-Met">
-                <span class="current-panel"></span>/
-                  <span class="total-panels">3</span>
-              </span>
-              <button class="panel-nav next-panel" data-direction="1" aria-label="Go to next panel"></button>
-            </div>
-          </onboarding-panel>
-          <footer>
-          <a class="button new-tab close-popup-after-click footer-button dashboard-link" href="">Manage Relay Addresses</a>
+          <div class="onboarding-pagination">
+            <button class="panel-nav previous-panel" data-direction="-1" aria-label="Show previous panel"></button>
+            <span class="panel-num fx-relay-font-met">
+              <span class="current-panel"></span>/
+                <span class="total-panels">3</span>
+            </span>
+            <button class="panel-nav next-panel" data-direction="1" aria-label="Go to next panel"></button>
```

Comment:
Q: Why is it "Show previous panel" vs "Go to next panel"?
Shouldn't it be "Go to previous panel" and "Go to next panel", or "Show previous panel" and "Show next panel" for consistency?